packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -266,6 +267,11 @@ class DropdownMenu<T> extends StatefulWidget {
   /// Defaults to true as the search function could be commonly used.
   final bool enableSearch;
 
+  /// The type of keyboard to use for editing the text.
+  ///
+  /// Defaults to [TextInputType.text].
```

Comment:
Nit: Maybe enforce this by setting a real default at line 162? Up to you.

Reply:
Or if you don't do this, add a test that verifies the default value that textField.keyboardType ends up with when keyboardType is not passed in DropdownMenu.

Reply:
I have added the additional test for keyboardtype default